app/src/main/java/com/r5k/contacerveja/util/CommonUtil.kt

```diff
@@ -1,24 +1,7 @@
 package com.r5k.contacerveja.util
 
-import android.app.ProgressDialog
-import android.content.Context
-import android.graphics.Color
-import androidx.core.graphics.drawable.toDrawable
-import com.r5k.contacerveja.R
 
 object CommonUtil {
 
-    fun showLoadingDialog(context: Context?): ProgressDialog {
-        val progressDialog = ProgressDialog(context)
-        progressDialog.let {
-            it.show()
-            it.window?.setBackgroundDrawable(Color.TRANSPARENT.toDrawable())
-            it.setContentView(R.layout.progress_dialog)
-            it.isIndeterminate = true
-            it.setCancelable(false)
-            it.setCanceledOnTouchOutside(false)
-            return it
-        }
-    }
```

Comment:
Same as MVPView.

Reply:
This we can remove, but probably will be used in the future.  Normally we have something to be extracted to utils class.